README.md

```diff
@@ -15,9 +15,13 @@ For more information on how to use the benchmark tool and configure it to your n
 Before building, you need to have a local copy of the [Aerospike C Client](https://github.com/aerospike/aerospike-client-c) and to have built it. After this, set the environment variable `CLIENTREPO` to point to the directory containing the built C client.
 
 Additional external dependencies:
- * libyaml
+ * libyaml-devel
```

Comment:
Depends on the distro - this would be for Redhat based distros, for Debian it would be libyaml-dev.